misk/web/tabs/config/package.json

```diff
@@ -14,17 +14,17 @@
     "test": "echo no test"
   },
   "dependencies": {
-    "@misk/common": "^0.0.70",
-    "@misk/components": "^0.0.90"
+    "@misk/common": "^0.1.0-alpha-0.2",
+    "@misk/components": "^0.1.0-alpha-0.7"
```

Comment:
This versioning seems strange. How come we're switching now?

Reply:
Previously we were releasing new minor versions for certain testing as features got developed knowing that they weren't long term or stable releases. The alpha builds will help signify whether a build is stable or is being used for active development.

Ie. 
`0.1.0` is stable release
`0.1.0-alpha-0.3` is in active development and may be partially broken